library/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Should it be under `/sample`, but not `/library`?

Reply:
Moved to sample